src/Service/FileUploader.php

```diff
@@ -25,4 +25,15 @@ public function upload(File $file): string
 
         return $filename;
     }
+
+    public function get(string $fileName): string
+    {
+        return $this->filesystem->read($fileName);
+    }
+
+    public function makeTempFile(string $fileName): void
+    {
+        $content = $this->get($fileName);
+        file_put_contents(sys_get_temp_dir().'/'.$fileName, $content);
```

Comment:
Remove `file_put_contents` call, and keep content in memory rather, until it goes in the filesystem using the filesystem interface.

Reply:
Issue #84 opened, do not fix here.